src/App.tsx

```diff
@@ -15,47 +15,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type IsReserved = boolean;
```

Comment:
The type alias 'IsReserved' is not necessary. You can use 'boolean' directly in the function signature for 'getReorderedGoods'.